ui/src/containers/CreateVolume.js

```diff
@@ -711,7 +732,7 @@ const CreateVolume = (props) => {
                           />
                         </SizeUnitFieldSelectContainer>
                       </SizeFieldContainer>
-                    ) : (
+                    ) : values.type === RAW_BLOCK_DEVICE ? (
```

Comment:
Now that there is more than 2 options I suggest to refactor this to a switch case